tests/rule_based_profiler/parameter_builder/test_numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -312,16 +312,12 @@ def test_oneshot_numeric_metric_range_multi_batch_parameter_builder_bobby(
 
     actual_value_05_lower: float = actual_values_05[0]
     actual_value_05_upper: float = actual_values_05[1]
-    expected_value_05_lower: float = -50.5
-    expected_value_05_upper: float = -21.1
+    expected_value_05_lower: float = -52.0
```

Comment:
@NathanFarmer A little strange since nothing other the name change...

Reply:
This is still WIP. You won't see this in the version that is ready.